pixeltable/globals.py

```diff
@@ -20,11 +20,74 @@
 
 _logger = logging.getLogger('pixeltable')
 
-
 def init() -> None:
     """Initializes the Pixeltable environment."""
     _ = Catalog.get()
 
+def _get_or_drop_existing_path(
+    path_str: str,
+    caller_context: catalog.SchemaObjectType,
+    if_exists: catalog.IfExistsParam
+) -> Optional[catalog.SchemaObject]:
+    """Handle existing dir, table, view, or snapshot, during create call as per user directive.
+
+    Args:
+        path_str: An existing and valid path to the dir, table, view, or snapshot.
+        caller_context: Whether the caller of this function is creating a dir, table, view, or snapshot, at the existing path.
+        if_exists: Directive regarding how to handle the existing path.
+            Must be one of the following:
```

Comment:
Not needed, this is not a user-facing API function. Also, if_exists is not a string parameter.

IfExistsParam is self-explanatory.

Reply:
Done.
<!-- Reviewable comment -OEl22d60QmVdsLmJkqD:b-896fix -->
<!-- Sent from Reviewable.io -->